R/get_biomart_pfam_merge.R

```diff
@@ -0,0 +1,120 @@
+#' Given pfam id(s) and an ensembl mart
+#' creates a grange for the pfam domains
+#' 
+#' @param ensembl a S4 class from biomaRt::useMart() call
```

Comment:
I think we should define this as a `Mart` object - and probably rename the parameter for clarity e.g. to `mart`